package.json

```diff
@@ -6,6 +6,10 @@
   "main": "lib/index.js",
   "scripts": {
     "test": "mocha test/index.js",
+    "test:ci": "yarn test && yarn test:ci:graphql-v15.x && yarn test:ci:graphql-v16.0 && yarn test:ci:graphql-v16.x",
```

Comment:
I believe we can get rid of the initial `yarn test` here, right?

I get the feeling we're going to be running the exact same tests as `yarn test:ci:graphql-v16.x`, no?

Reply:
The naked `yarn test` was testing `15.0.0`, but I definitely feel this should be made more explicit by pinning this version if `package.json` too.

I probably should have mentioned in my annotated comments, but I left the library using v15 since that now has more active tests, and so it helps for that to be the thing that developers are running locally.

Reply:
Ahhh, gotcha! I suppose that makes sense for now.

Longer term, perhaps it'd be good to use latest `graphql` in `devDependencies`, that way we're always working against the latest version, but would still have test coverage for older versions.